recipe/meta.yaml

```diff
@@ -45,8 +50,8 @@ requirements:
     - python                              # [build_platform != target_platform]
     - cython                              # [build_platform != target_platform]
   host:
-    - cuda-cudart-dev
-    - cuda-nvrtc-dev
+    - cuda-cudart-dev    # cuda-cudart run-export is ignored, see above
+    - cuda-nvrtc-dev     # cuda-nvrtc run-export is ignored, see above
```

Comment:
I think these comments are more useful to place in the run/run_constrained sections since the effect of ignoring of run exports is that we have to manually include the corresponding runtime requirements in the run/run_constrained.

Reply:
Done in https://github.com/conda-forge/cuda-python-feedstock/pull/67/commits/64591d863fddd4ed766bf37b54bd2cffd7051442